api/src/messages/services/messages.service.ts

```diff
@@ -21,22 +21,13 @@ export class MessagesService {
     }
 
     async findAll() {
-        return this.messageRepository.find({
-            relations: {
-                author: true
-            }
-        });
+        return this.messageRepository.find();
     }
 
     async findOne(id: number) {
-        return this.messageRepository
-            .findOneOrFail({
-                where: { id },
-                relations: { author: true }
-            })
-            .catch(() => {
-                throw new NotFoundException();
-            });
+        return this.messageRepository.findOneByOrFail({ id }).catch(() => {
+            throw new NotFoundException();
```

Comment:
1. 100% NotFoundException? Even when other error occure? 
2. This thorw NotFoundException and in method below you catch error and throw UnauthorizedException. So think about whether you want to throw one or the other.

Reply:
I know about it, but that's on 99% not found, because on development, i can have other error, but after that, that's no way to have other error, don't think about it.

Reply:
Done, error handling work correctly now.